iulianIachimov/src/main/java/datacollectiontask/Main.java

```diff
@@ -0,0 +1,123 @@
+package datacollectiontask;
+
+import datacollectiontask.Models.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        //create Addresses (2 possibilities)
+        //1
+        Address homeAddressOne = new Address("UK", "Liverpool", "89 Baker Str.");
+        Address workAddressOne = new Address("UK", "Bornemouth", "15 King Str.");
+
+        Map<String, Address> addressMapOne = new HashMap();
+        addressMapOne.put("Home", homeAddressOne);
+        addressMapOne.put("Work", workAddressOne);
+
+        //2
+        Map<String, Address> addressMapTwo = new HashMap();
+        addressMapTwo.put("Home", new Address("Republic of Moldova", "Chisinau", "12 Dacia Str."));
+        addressMapTwo.put("Work", new Address("Republic of Moldova", "Chisinau", "45/3 Hotin Str."));
+
+        //Creating users
+        User userOne = new User("Jill Jackson", addressMapOne);
+        User userTwo = new User("Adam Smith", addressMapTwo);
+
+        // create first list of items
+        List<Item> ItemListOne = new ArrayList<>();
+        Item itemOne = new Item("Iphone", Type.GADGET);  // create itemOne
+        Item itemTwo = new Item("Trousers", Type.CLOTHES); // create itemTwo
+        ItemListOne.add(itemOne); // add itemOne to ItemListOne
+        ItemListOne.add(itemTwo);
+
+        //Create second list of items
+        List<Item> ItemListTwo = new ArrayList<>();
+        Item itemThree = new Item("Trotinette", Type.TOY);  // create itemOne
+        Item itemFour = new Item("Frying Pan", Type.KITCHENWARE); // create itemTwo
+        ItemListTwo.add(itemThree); // add itemOne to ItemListOne
+        ItemListTwo.add(itemFour);
+
+        //Create 2 objects of Order type ()
+        Order orderOne = new Order(userOne, ItemListOne);
+        Order orderTwo = new Order(userTwo, ItemListTwo);
+
+        //Create list of orders
+        List<Order> orderList = new ArrayList<>();
+        orderList.add(orderOne);
+        orderList.add(orderTwo);
+
+        //Filter Orders by Type CLOTHES, calling funcion filterOrderByItemType(orderList)
+        List<Order> filteredOrderList = filterOrderByItemType(orderList);
+        System.out.println("the size of filtered order list is " + filteredOrderList.size());
+
+        /*
```

Comment:
The staff which is done out of task could be stored out of the package, I would suggest to create a separate package within your directory e.g. practice -> where you can place the similar things

Reply:
Does this separate package need to be commited to origin as well, or we can keep it for ourselves?